tests/func/test_import_url.py

```diff
@@ -90,15 +89,16 @@ def test(self):
 
 
 @pytest.mark.parametrize("dname", [".", "dir", "dir/subdir"])
-def test_import_url_to_dir(dname, repo_dir, dvc_repo):
-    src = repo_dir.DATA
+def test_import_url_to_dir(dname, tmp_dir, dvc):
+    tmp_dir.gen({"data_dir": {"file": "file content"}})
+    src = os.path.join("data_dir", "file")
 
     makedirs(dname, exist_ok=True)
 
-    stage = dvc_repo.imp_url(src, dname)
+    stage = dvc.imp_url(src, dname)
 
-    dst = os.path.join(dname, os.path.basename(src))
+    dst = os.path.join(dname, "file")
```

Comment:
Minor comment:
```suggestion
    dst = tmp_dir / dname / "file"
```
and then you could
```
```
stage.outs[0] == dst 
```
and so on.